### PR TITLE
Add H801-mqtt-json implementation

### PR DESCRIPTION
**Pull request by starkillerOG**
**Description:**
I thought it might be nice to include my firmware in the implementations section if other people want to use my firmware.
@emontnemery also recommended me to add a link.

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#27253

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by starkillerOG** [comment]
Alright this finally looks like a correct PR.
Sorry for the mess of the two previous PRs that I closed:
https://github.com/home-assistant/home-assistant.io/pull/10762
https://github.com/home-assistant/home-assistant.io/pull/10761

File: source/_integrations/light.mqtt.markdown [hunk 1]
```diff
@@ -650,6 +650,8 @@ light:
 
 - [AiLight](https://github.com/stelgenhof/AiLight) is a custom firmware for the Ai-Thinker (and equivalent) RGBW WiFi light bulbs that has an ESP8266 onboard and controlled by the MY9291 LED driver. It implements the [MQTT JSON light](/integrations/light.mqtt) platform and supports ON/OFF, RGBW colours, brightness, colour temperature, flashing and transitions. Also it includes [MQTT Auto Discovery](/docs/mqtt/discovery/)) and the MQTT Last Will and Testament is enabled as well.
 
+- [h801-mqtt-json](https://github.com/starkillerOG/h801-mqtt-json) is a custom firmware for the H801 LED dimmer, a 5 channel (RGBWWW)  WiFi LED strip controller for 12V LED strips. The firmware is meant to control the 5 channels of the H801 to simultaneously control an RGB and a Warm-white/Cold-white Led strip such as an 5050 RGB LED strip and a 5025 Dual White strip. It implements the [MQTT JSON light](/integrations/light.mqtt) platform and supports ON/OFF, RGBW colours (RGB strip), brightness, color temperature (WWW strip) and transitions.
```
Comment (emontnemery) [inline source/_integrations/light.mqtt.markdown]: Change "WWW strip" to "CW/WW strip" or "CW strip".

Comment (starkillerOG) [reply source/_integrations/light.mqtt.markdown]: done

**Review by emontnemery — APPROVED** [review]

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Review by frenck — APPROVED** [review]
Thanks, @starkillerOG! 👍 

Happy Hacktoberfest 🎉

**Pull request merged**